docs/database-classes.md

```diff
@@ -277,8 +277,41 @@ transactor.Transact(db =>
 });
 ```
 
+## Discovery
+
+Starcounter automatically discovers all database classes in the application assembly, which does not include all referenced assemblies. Manual configuration is required to use database classes from referenced assemblies.
+
+### Referenced assembly
```

Comment:
```suggestion
### Referenced assembly
```

Personally, I would not use headings for this illustration. Instead, describe the separation in body text.

 ```suggestion

### Example

The `Something` database class definition below is contained in an assembly that our app references:

```